velox/functions/prestosql/tests/TypeOfTest.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/functions/prestosql/tests/FunctionBaseTest.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::test;
+
+class TypeOfTest : public functions::test::FunctionBaseTest {};
+
+TEST_F(TypeOfTest, noNulls) {
```

Comment:
Let's add tests for nulls, all primitive and complex types. See presto-main/src/test/java/com/facebook/presto/operator/scalar/TestTypeOfFunction.java for the tests cases in Presto Java.